content/200-concepts/100-components/02-prisma-client/050-filtering-and-sorting.mdx

```diff
@@ -323,7 +323,7 @@ const getActiveUsers = await prisma.user.findMany({
 
 ### Sort by relevance (PostgreSQL)
 
-In [3.5.0](https://github.com/prisma/prisma/releases/3.5.0) and later, when using PostgresSQL you can sort records by relevance to the query using the `_relevance` keyword. This uses the relevance ranking functions from PostgreSQL's full text search feature, which are explained further in [the PostgreSQL documentation](https://www.postgresql.org/docs/12/textsearch-controls.html).
```

Comment:
Interesting why cSpell did not warn about this already... that is why we have a spell checker.

Reply:
It may be likely that cSpell accepts both spellings and isn't as strict as we(Prisma) are on terminology 🤔